app/serializers/lib_search_result_serializer.rb

```diff
@@ -0,0 +1,21 @@
+class LibSearchResultSerializer < ObjectSerializer
+  set_id { |object| Fingerprintable::Fingerprinter.new(object: object).fingerprint }
+
+  attributes :pagination, :query
+
+  attribute :ledger do |_object, params|
+    params[:ledger].id
+  end
+
+  attribute :next_page_pagination do |object|
+    object.next_searcher.try(:pagination)
+  end
```

Comment:
How do I interpret this?
```
      "next_page_pagination": {
        "limit": 10,
        "offset": 11
      },
```
Does that mean the next search results start at item 11? Also, does this necessarily mean there are more results beyond where we're currently at?

Reply:
See my PR comment and this commit: https://github.com/LedgerSync/rails_app/pull/39/commits/6f9ac04d3f8378b274632db5b610773875e27618